### PR TITLE
Removed Ambulance button

### PR DESCRIPTION
**Pull request by dmitrii-zolotukhin**
fixed sparkletown/internal-sparkle-issues#1154

### CODE REVIEW

**Comment by alexey-petrov** [comment]
Now that the only component (`SparkleFairiesPopUp`) that depends on `showRangers` prop is removed are we going to remove all the mentions of `showRangers` and related toggle buttons/props as well? @dmitrii-zolotukhin

File: src/components/templates/ArtPiece/ArtPiece.tsx [hunk 1]
```diff
@@ -35,8 +34,7 @@ export interface ArtPieceProps {
 
 export const ArtPiece: React.FC<ArtPieceProps> = ({ venue }) => {
   // NOTE: venue should always be there, but per the if(!venue) check bellow, better make safe than sorry
-  const { name, host, config, showRangers, iframeUrl, videoAspect } =
-    venue ?? {};
+  const { name, host, config, iframeUrl, videoAspect } = venue ?? {};
```
Comment (sunny-viktoryia) [inline src/components/templates/ArtPiece/ArtPiece.tsx:37]: looks like `showRangers` is a legacy feature. Please, remove it everywhere from the code.

Comment (dmitrii-zolotukhin) [reply src/components/templates/ArtPiece/ArtPiece.tsx:37]: @sunny-viktoryia fixed in d4dcd6783fd04b2501d6998a02a6d4f38db51acb

**Comment by dmitrii-zolotukhin** [comment]
> Now that the only component (`SparkleFairiesPopUp`) that depends on `showRangers` prop is removed are we going to remove all the mentions of `showRangers` and related toggle buttons/props as well? @dmitrii-zolotukhin

fixed in d4dcd6783fd04b2501d6998a02a6d4f38db51acb

**Review by sunny-viktoryia — APPROVED** [review]
Looks good to me. 🙃 

Please, also remove 4 consts from the `settings`
`PLAYA_INFO_URL`
`PLAYA_INFO_NAME`
`REALITY_RANGERS_URL`
`REALITY_RANGERS_NAME`

because we don't use them in the project anymore. And then the PR can be merged. 🙃 
Thanks!

**Pull request merged**